### DISCUSSION TITLE
redball++

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#700] [Q&A] [answered]
Hi,

I am interested in implementing RedBall++ (https://www.youtube.com/watch?v=NMzhDqVgVvk) on my iCub. Since I recently acquired an Intel D435i depth camera, I believe it is necessary to deploy the icub-gazebo-grasping on the iCub laptop platform which i already completed. Should I start by performing the [realsense-holder-calibration](https://github.com/robotology/realsense-holder-calibration/) first?

Additionally, I am seeking documentation or information about RedBall++ beyond the video.

**Comment by pattacini** [comment, marked as answer]
Hi @jimingda 

The so-called RedBall++ is hosted at this repository:
- https://github.com/hsp-iit/roft.

ROFT is the name of the involved pipeline for 6DOF Pose Estimation. That repository contains all the information you need.

The main author is @xEnVrE, who recently left IIT though. At any rate, the lab he was with is [HSP@IIT](https://hsp.iit.it/), led by @lornat75.

> believe it is necessary to deploy the icub-gazebo-grasping on the iCub laptop platform which i already completed

Nope, it is not necessary; just rely on ROFT.

>Should I start by performing the [realsense-holder-calibration](https://github.com/robotology/realsense-holder-calibration/) first?

That's one of the main step, indeed.
